sphinxgallery/docs_resolv.py

```diff
@@ -409,6 +409,8 @@ def embed_code_links(app, exception):
     """Embed hyperlinks to documentation into example code"""
     if exception is not None:
         return
+    if not app.builder.config.plot_gallery:
+        return  # no need to embed
```

Comment:
@lesteve please take over. I am mostly on my phone these days. It will be
faster if you do a cosmit on your side. Je te donne carte blanche :)